ui/index.html

```diff
@@ -104,7 +104,7 @@ <h1>Stable Diffusion UI <small>v2.3.2 <span id="updateBranchLabel"></span></smal
             <div class="line-separator">&nbsp;</div>
 
             <div id="editor-settings" class="panel-box settings-box">
-                <h4 class="collapsible">Image Settings</h4>
+                <h4 class="collapsible">Image Settings<i id="reset-image-settings" class="fa-solid fa-arrow-rotate-left"></i></h4>
```

Comment:
Can we have a tooltip displayed over this? Or change it to a text button saying "Restore Defaults"?

Reply:
Sure! I've added a tooltip. Not sure how we want to do tooltips so i copied some tooltip css from one of my other projects, and we can change it as we like in the future. If you are looking for some more specific styling for tooltips let me know and i can update it.